18F/dol-whd-14c
# Pa11y npm package and Microsoft.VisualStudio.QualityTools.UnitTestFramework references

#### What's this PR do?
1. Adds npm package for pa11y to dev dependencies
2. Fixes Microsoft.VisualStudio.QualityTools.UnitTestFramework reference within DOL.WHD.Section14c.Test Project to reference local package folder (in root directory)

#### Where should the reviewer start?
1. package.json in DOL.WHD.Section14c.Web project
2. Check that Microsoft.VisualStudio.QualityTools.UnitTestFramework reference in DOL.WHD.Section14c.Test project is being referenced within the packages folder in the root folder of the project

#### How should this be manually tested?
1. Verify pa11y is listed in package.json
2. Clone and build Visual Studio API project and verify successful build

#### Any background context you want to provide?
#### What are the relevant tickets?

This pull request will fix MSBuild not being able to locate Microsoft.VisualStudio.QualityTools.UnitTestFramework reference
#181
 
#### Screenshots (if appropriate)
#### Questions:
